xmcloud.build.json

```diff
@@ -6,8 +6,6 @@
         "jssDeploymentSecret": "110F1C44A496B45478640DD36F80C18C9",
         "enabled": true,
         "type": "sxa",
-        "lintCommand": "lint",
```

Comment:
@dma-sitecore Should `lintCommand` be removed? It's still [in the docs](https://doc.sitecore.com/xmc/en/developers/xm-cloud/the-xm-cloud-build-configuration.html#:~:text=%22jss%22.-,lintCommand,-string).